Library/Formula/libxdg-basedir.rb

```diff
@@ -0,0 +1,30 @@
+class LibxdgBasedir < Formula
+  homepage "https://github.com/devnev/libxdg-basedir"
+  url "http://github.com/devnev/libxdg-basedir/archive/libxdg-basedir-1.2.0.tar.gz"
+  sha256 "1c2b0032a539033313b5be2e48ddd0ae94c84faf21d93956d53562eef4614868"
+  desc "An implementation of the XDG Base Directory specifications"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+
+  def install
+    system "./autogen.sh", "--disable-dependency-tracking",
```

Comment:
:+1:. It'd be nice of upstream to release tarballs with `configure` scripts pre-generated so we can drop the autotool deps, but I don't have an issue with this setup.